dropwizard-example/src/main/java/com/example/helloworld/core/Person.java

```diff
@@ -14,13 +13,10 @@
 
 @Entity
 @Table(name = "people")
-@NamedQueries(
```

Comment:
@joschi this look ok to you?

Reply:
This came up as a linter thing - it said prior to Java 8, you couldn't repeat an annotation multiple times, hence the creation of annotations which simply served as container annotations.
Java 8+ supports specifying multiple annotations of the same type, so `NamedQueries` et al are no longer required.
I believed the linter, but agree it would be good to get someone who actually knows to confirm :)

Reply:
A little more info - here's an Oracle doc introducing repeatable annotations in Java 8: https://docs.oracle.com/javase/tutorial/java/annotations/repeating.html

The JPA 2.2 spec is quite hilariously brief: https://jakarta.ee/specifications/persistence/2.2/persistence_2.2.html

I haven't found the actual JPA announcement supporting them, but here's a blog post talking about it: https://thorben-janssen.com/jpa-2-2-repeatable-annotations/

and here's one about Hibernate 5.3 supporting it: https://in.relation.to/2018/02/07/hibernate-53-repeating-jpa-annotations/